[Workers] pricing revision (#2740)
cloudflare/cloudflare-docs: products/workers/src/content/platform/pricing.md

# Pricing

> By default, users have access to the Workers Free plan. The Workers free plan includes limited usage of Workers and Workers KV. Read more about the [Free plan limits](/platform/limits#worker-limits).

Reviewer: ```suggestion
By default, users have access to the Workers Free plan, which includes limited usage of Workers and Workers KV. Read more about the [Free plan limits](/platform/limits#worker-limits).
```

  
</TableWrap>

1. Cloudflare will bill for duration charges based on the higher of your wall time or CPU time, with a multiple applied to the CPU time to account for the processing power allotted to your script. Cloudflare will not bill for wall time duration charges beyond the execution [limit](/platform/limits#worker-limits) given.
2. Duration billing charges for the 128 MB of memory your Worker is allocated, regardless of actual usage. If your account has significant traffic to a single Worker, multiple instances of that Worker may run in the same isolate on the same physical machine and share the 128 MB of memory. These Workers are still billed as if they are allocated a full 128 MB of memory.

### Usage models

Workers are available under two Usage Models: Bundled and Unbound. Usage Models are settings on your Workers that specify the upper [limits](/platform/limits) for how long a Worker can execute. In addition to different limits, Workers on the Bundled Usage Model have usage billing based on requests only, while Workers on Unbound have usage billing based on requests and duration at the rates shown under [pricing](/platform/pricing#pricing).

#### Default usage model

> The default Usage Model is used when new Workers are created, and no Usage Model is provided via configuration during creation. When an account is first upgraded to the Paid plan, the default Usage Model is set to Unbound. The default can be changed at any time by using the Default Usage Model setting on the Workers overview page. Cloudflare recommends setting the default to the type of Worker you create the most. Existing Workers will not be impacted when changing the default Usage Model.

Author: @lukeed "via configuration during creation"

> The default Usage Model is used when new Workers are created, and no Usage Model is provided via configuration during creation. When an account is first upgraded to the Paid plan, the default Usage Model is set to Unbound. The default can be changed at any time by using the Default Usage Model setting on the Workers overview page. Cloudflare recommends setting the default to the type of Worker you create the most. Existing Workers will not be impacted when changing the default Usage Model.

Reviewer 2: @deadlypants1973 You likely have to adjust my UI step formatting haha. All the names are correct / as-written in UI

```suggestion
When an account is first upgraded to the Paid plan, the Unbound plan is used as the default Usage Model. You may change your default Usage Model account-wide by going to the **Cloudflare Dashboard** > **Workers** > **Overview** and **Default Usage Model** > **Change** in the sidebar. Cloudflare recommends setting the default to the type of Worker you create the most. Existing Workers will not be impacted when changing the default Usage Model.

You may change the Usage Model for individual Workers without affecting your account-wide default. This may be done through the [`usage_model` key](https://developers.cloudflare.com/workers/cli-wrangler/configuration#keys) in your `wranger.toml` file or through the dashboard: **Workers** > **select your worker** > **Settings** > **Usage Model**.
```

  
</TableWrap>

1. Cloudflare will bill for duration charges based on the higher of your wall time or CPU time, with a multiple applied to the CPU time to account for the processing power allotted to your script. Cloudflare will not bill for wall time duration charges beyond the execution [limit](/platform/limits#worker-limits) given.

> 2. Duration billing charges for the 128 MB of memory your Worker is allocated, regardless of actual usage. If your account has significant traffic to a single Worker, multiple instances of that Worker may run in the same isolate on the same physical machine and share the 128 MB of memory. These Workers are still billed as if they are allocated a full 128 MB of memory.

Reviewer: ```suggestion
2. Duration billing will charge for the 128 MB of memory allocated to your Worker, regardless of actual usage. If your account has significant traffic to a single Worker, multiple instances of that Worker may run in the same isolate on the same physical machine and share the 128 MB of memory. These Workers are still billed as if they are allocated a full 128 MB of memory.
```
I almost couldn't see the verb in the sentence, and I think this change makes it more readable.

### Usage models

Workers are available under two Usage Models: Bundled and Unbound. Usage Models are settings on your Workers that specify the upper [limits](/platform/limits) for how long a Worker can execute. In addition to different limits, Workers on the Bundled Usage Model have usage billing based on requests only, while Workers on Unbound have usage billing based on requests and duration at the rates shown under [pricing](/platform/pricing#pricing).

#### Default usage model

When an account is first upgraded to the Paid plan, the Unbound plan is used as the default Usage Model. You may change your default Usage Model account-wide by going to the **Account Home** > **Workers** > **Overview** > **Default Usage Model** > **Change**. Cloudflare recommends setting the default to the type of Worker you create the most. Existing Workers will not be impacted when changing the default Usage Model.

> You may change the Usage Model for individual Workers without affecting your account-wide default. This may be done through the [`usage_model` key](https://developers.cloudflare.com/workers/cli-wrangler/configuration#keys) in your `wranger.toml` file or through the dashboard: **Workers** > **select your Worker** > **Settings** > **Usage Model**.

Reviewer: ```suggestion
You may change the Usage Model for individual Workers without affecting your account-wide default. You can do this through the [`usage_model` key](https://developers.cloudflare.com/workers/cli-wrangler/configuration#keys) in your `wranger.toml` file or through the dashboard: **Workers** > **select your Worker** > **Settings** > **Usage Model**.
```
